tests/infra/ccf.py

```diff
@@ -462,22 +462,27 @@ def find_primary_and_any_backup(self, timeout=3):
     def wait_for_all_nodes_to_catch_up(self, primary, timeout=3):
         """
         Wait for all nodes to have joined the network and globally replicated
-        all transactions executed on the primary (including the transactions
+        all transactions globally executed on the primary (including transactions
```

Comment:
I don't think this comment is correct. The body of `getCommit` returns `tables->commit_version`, which is the node's local commit version (the _global_ commit is retrieved through `consensus->get_commit_seqno`). I think the logic is still fine (the nodes can 'catch up' with a target index whether its a global commit point or not), but the comment is misleading.

I think this is a strong argument for renaming some of the kv concepts - we already have duplicated consensus terms, but if we're talking about transactions being _committed_ at a _seqno_ in the _consensus_, we shouldn't mistake this with the _version_ where they're _applied_ to the _kv_.

Reply:
`tables->commit_version()` returns the `compacted` version which is the global commit version? I got caught by this too originally, hence the retry logic around this as the start node may not have anything globally committed when it first starts up.

Reply:
Ah ha, you're absolutely right (I was looking at `TxView::commit_version()`, but we're calling `Store::commit_version()`). So this _is_ a global commit point.